src/components/Post/EditPostComponent.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import { Segment } from 'semantic-ui-react';
+
+class EditPost extends Component {
+  render() {
+    return (
```

Comment:
Seems like `EditPost` only renders `jsx`. If so, this should be a functional dummy component, but at the same time, I envision this having its own functionality with its own states. Is that correct?

Reply:
yup

Reply:
Gotcha. Then I'm fine with this.